recipe/common.php

```diff
@@ -18,13 +18,21 @@
 set('clear_paths', []);         // Relative path from deploy_path
 set('clear_use_sudo', true);    // Using sudo in clean commands?
 
+
+/**
+ * Composer options
+ */
+env('composer_action', 'install');
+env('composer_options', '{{composer_action}} --verbose --prefer-dist --no-progress --no-interaction --no-dev --optimize-autoloader');
```

Comment:
Crosses fingers... I understand your gut-reaction against running update ad-hoc, but it can be useful for automated deployments to dev server. `compoaser_options` already exists, so it's easy enough to overwrite the whole string, but this just pulls out `composer_action`. Give me a heads up if this is okay @oanhnn and @elfet .